pkg/util/net.go

```diff
@@ -38,12 +37,6 @@ const (
 )
 
 func GetHostIP() (string, error) {
-	ip, err := net.ChooseHostInterface()
-	if err == nil {
-		return ip.String(), nil
-	}
-	klog.V(2).Infof("failed to find default route IP address: %v", err)
-
```

Comment:
We may want to keep this logic when getOVNGatewayIP fails to support other CNIs.

Reply:
also note that we are discussing to remove plugin specific code from microshift main process through the [pluggable plugin enhancement](https://github.com/openshift/enhancements/pull/1304/),  I wonder if we should let microshift to detect the node IP here and move the configure-ovs.sh execution plugin specific code.
/cc @mangelajo

Reply:
I don't understand why this change forces the testing into having a fully populated config structure. How is it related?

Reply:
When we run the test, there's no OVN-K, thus bridge `br-ex` doesn't exist. So `GetHostIP` will fail.